src/enkf/gridio_gfs.f90

```diff
@@ -4094,45 +4314,157 @@ subroutine writeincrement_pnc(vars3d,vars2d,n3d,n2d,levels,ndim,grdin,no_inflate
   call nccheck_incr(nf90_put_var(ncid_out, o3varid, sngl(inc3dout), &
                       start = ncstart, count = nccount))
 
+  ! For hydrometeors, following the treatment for specific humidity increment
+  ! Need to make sure the analysis value is not negative
+  ! Read in background + increment and make sure the minimum is qcmin
+  ! Adjust increment accordingly
+  ! ToDo #1: check o3, should do the same
+  ! ToDo #2: add handling for Thompson scheme (imp_physics: 8) 
```

Comment:
ToDo notes should not be in code merged into develop.  ToDo should be addressed in a future PR.